.gitignore

```diff
@@ -28,6 +28,7 @@
 .idea/**/workspace.xml
 .idea/**/tasks.xml
 .idea/dictionaries
+.idea/topics-driven-backlog.iml
```

Comment:
If we don't want the rubymine project to be included please include the entire folder (and clean up the selective gitignore) ;)

Reply:
I am not really sure, whether we need the .iml file in here. Do we? Would be nice to have the ide setup in version controll.

Reply:
we do need the file for the IDE setup. the gitignore I copied in is the "official" one suggested by Jetbrains for their IDEs, keeping all relevant files but leaving out user specific ones (like the workspace.xml).

Reply:
so then, I will remove the line.